### DISCUSSION TITLE
Snapdrop

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#2005] [General]
I recently installed the Bromite SystemWebView, and I am sad too see that one of my fairly often used app/site does not work (https://snapdrop.net/).
  Devices are detected, just files can't be sent.
  Is it just a bug, or a cause of the privacy/security enhancements making this the intended behavior? (I don't see the reason why would it be intentional)
  Will it be fixed, or there are no plans for that?

**Comment by csagan5** [comment]
Discussions cannot be used to submit bugs.

> Will it be fixed, or there are no plans for that?

I suggest to search on the issue tracker before submitting a bug or starting a discussion.
